apps/widget-configurator/src/app/analytics.ts

```diff
@@ -0,0 +1,22 @@
+import { sendEvent, Category } from '@cowprotocol/analytics'
+
+export function connectWalletToConfigurator() {
+  sendEvent({
+    category: Category.WIDGET_CONFIGURATOR,
+    action: 'Connect wallet',
+  })
+}
+
+export function viewEmbedCode() {
+  sendEvent({
+    category: Category.WIDGET_CONFIGURATOR,
+    action: 'View embed code',
```

Comment:
should we remove the embed word.

Also, should be embedded? right? Anyways, IMO "View code" is enough

Reply:
Fixed